src/partest/scala/tools/partest/nest/Runner.scala

```diff
@@ -462,14 +459,11 @@ class Runner(val testInfo: TestInfo, val suiteRunner: AbstractRunner) {
     state
   }
 
-  // snort or scarf all the contributing flags files
+  // all sources in a round may contribute flags via // scalac: -flags
   def flagsForCompilation(sources: List[File]): List[String] = {
-    val perKind  = readOptionsFile(testFile.getParentFile.changeExtension("flags"))
-    val perTest  = readOptionsFile(flagsFile)
```

Comment:
Is there really no demand for per-test ("par-test", it's in a name) flags? Other than from me, of course. Scala isn't a one-and-only-one-way-to-do-it language.

Reply:
Besides the sheer drudgery of moving the options into sources, I had expected to find cases where shared flags were very desirable -- there were maybe 2 or 3 tests where options applied to more than one "source set", I think just optimizer flags. For example, in the many macros tests, only the (usually) one file using the word macro needs the flag or import.